packages/ckeditor5-dev-tests/lib/utils/automated-tests/getkarmaconfig.js

```diff
@@ -174,3 +158,22 @@ module.exports = function getKarmaConfig( options ) {
 	return karmaConfig;
 };
 
+// Returns the value of Karma's browser option.
+// @returns {Array|null}
+function getBrowsers( options ) {
+	if ( process.env.TRAVIS ) {
+		return [ 'CHROME_TRAVIS_CI' ];
+	}
+
+	if ( options.server || !options.browsers ) {
```

Comment:
I just wanted to keep this: https://github.com/ckeditor/ckeditor5-dev/pull/166/files#diff-31df1471c2e1fbb9040249aaaf4d952aL47, but I changed the behaviour a bit during refactoring. That wasn't intentional, but it seems that I, indeed, handled `--browsers=''` :D